middlewares/errorHandler.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (err, req, res, next) => {
+    if (err.name === 'SequelizeValidationError') {
+        let error = [];
+        err.errors.forEach(x => {
+            error.push(x.message);
+        });
+        res.status(400).json(error);
+    } else if (err.name === 'SequelizeDatabaseError') {
+        res.status(500).json({
+            msg: 'Server Error'
```

Comment:
SequelizeDatabaseError gak semuanya menghasilkan status code 500, ada juga yang 400 jika error.parent.code === '23502' NOT NULL CONSTRAINT VIOLATION, jika codingan kamu memungkinkan untuk error ini terpanggil maka harus di handle juga